### ISSUE TITLE
In GPUImageMovieWriter createDataFBO destroyDataFBO memory leak

### ISSUE THREAD

**Issue by MuFengYi** [BradLarson/GPUImage#2522]

**Comment by MuFengYi** [comment]
<img width="933" alt="2e18be36-88a7-4f9e-a7d4-e1891164b2a1" src="https://user-images.githubusercontent.com/11486628/32158767-55c97c60-bd85-11e7-9495-8ecc0ee3fd09.png">

**Comment by MuFengYi** [comment]
#1942 The _movieWriterContext = [GPUImageContext sharedImageProcessingContext]; solution fixed the leak for me as well.

**Comment by eevictor** [comment]
Yes, #1942 The _movieWriterContext = [GPUImageContext sharedImageProcessingContext]; solution fixed the leak for me as well.
Without this fix, I could record up to 2 minutes, and the free memory drops from 1.6 GB to few MB and then crash.
